app/Http/Controllers/Api/CustomFieldsController.php

```diff
@@ -59,7 +59,13 @@ public function update(Request $request, $id)
     {
         $this->authorize('edit', CustomField::class);
         $field = CustomField::findOrFail($id);
-        $data = $request->all();
+        
+        /**
+         * Updated values for the field, 
+         * without the "field_encrypted" flag, preventing the change of encryption status
+         * @var array
+         */
+        $data = $request->except(['field_encrypted']);
```

Comment:
> It's not required, but I'd bet if someone includes it, it will allow them to (intentionally or not) change whether or not the field is encrypted, which could cause problems.

That should be prevented by this line, even if somebody includes `field_encrypted=1` on their request, it will be stripped from the `data` array, so the `$field->fill($data);` later would never contain the flag.

Reply:
Derp. Yep, you're right - I hadn't noticed that you included the API controller in there. (It's very late here, and I am a little dumb right now... )

Reply:
No problem! I made #5815 when it was like 3am here, I know the feeling! 😄